README.md

```diff
@@ -1,74 +1,116 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
-
-## Built With 
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available) 💻
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-You must have a browser on your device, visual code, a git version, [node.js](https://nodejs.org/en/) installed in your device and clone the repo from GitHub and follow the steps below.
-
-### Setup
-
-Clone the repository from GitHub, open the project folder, inside the project direcory open the commanline or Terminal and install the dependicies.
-
-### Install
-
-inside the project direcory open the commanline or Terminal, run npm install to install all the dependicies.
-
-### Usage
-
-To use this project or app run npm start in the project directory and you will see the live serever open on youe browser.
-
-### Run tests
-
-For running tests inside the project directory run npm test in command line or Terminal.
-
-### Deployment
-
-
-## Authors
-
-- GitHub: [@ahzamir](https://github.com/ahzamir).
-- Twitter: [@YousufiZamir](https://twitter.com/YousufiZamir)
-- LinkedIn: [ahmad-zamir-yousufi](https://www.linkedin.com/in/ahzamir/)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Budget App
+
+> This is a budget app that allows users to add their expenses categories and expenses. It also allows users to see their total expenses.
+
+## Live Demo
+- [Video demo](https://www.loom.com/share/30f987d8de024de8b90b6ace0016a6d0)
+- [Budget_app - Comming soon]()
+

```

Comment:
- [ ] Please make sure that your website is hosted and add the live demo link to your README.md file. As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Deploy%20the%20project%2C%20and%20test%20for%20final%20details.) requirement.

Reply:
@ShadyShawkat  I have an issue while deploying my app using Heroku. I am using WSL in windows.
This is the error I face while deploying.

![Screenshot 2022-09-22 171136](https://user-images.githubusercontent.com/96838030/191749845-7d6f0c8e-f84c-4c58-ada0-a3d4ca2e67b6.png)
![Screenshot 2022-09-22 171115](https://user-images.githubusercontent.com/96838030/191749877-349134cd-7649-4672-88cf-468a06d84e4a.png)

Reply:
@ShadyShawkat  Can you help me figure it out?